indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -161,6 +165,24 @@ public String apply(Instance input)
 
       log.info("Created instances: %s", instanceIds);
 
+      if (envConfig.getNodeData().getTags() != null) {
+        final List<Tag> tags = Lists.transform(
+            new ArrayList<>(envConfig.getNodeData().getTags().entrySet()),
```

Comment:
no need to create a new list, you can directly apply `Collections2.transform` on the entrySet and use the resulting collection on the `withTags` call.